### PR TITLE
Request type command constants

### PR DESCRIPTION
**Pull request by MyckolaPetrychko**

### CODE REVIEW

File: softcourse/basis/common-protocol/src/main/java/org/softgroup/common/protocol/RequestCommand.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by pvm on 07.04.17.
+ */
+public abstract class RequestCommand {
+
+    private List<String> commands = new ArrayList<>();
```
Comment (Dominys) [inline softcourse/basis/common-protocol/src/main/java/org/softgroup/common/protocol/RequestCommand.java]: @MyckolaPetrychko Why you need such strange classes ?

File: softcourse/services/authorization/authorization-api/src/main/java/org/softgroup/authorization/api/commands/AuthRequestCommand.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package org.softgroup.authorization.api.commands;
+
+import org.softgroup.common.protocol.RequestCommand;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by pvm on 07.04.17.
+ */
+@Component
+public class AuthRequestCommand extends RequestCommand {
```
Comment (Dominys) [inline softcourse/services/authorization/authorization-api/src/main/java/org/softgroup/authorization/api/commands/AuthRequestCommand.java]: @MyckolaPetrychko This is constants. Why you need component ?

Comment (MyckolaPetrychko) [inline softcourse/services/authorization/authorization-api/src/main/java/org/softgroup/authorization/api/commands/AuthRequestCommand.java]: In our frontend rest module we have to check if request has a valid type and command. Using this abstract classes and their child classes we can automatically fetch all available constants from all services and check if incoming request is valid.

Comment (Dominys) [inline softcourse/services/authorization/authorization-api/src/main/java/org/softgroup/authorization/api/commands/AuthRequestCommand.java]: @MyckolaPetrychko This is constants. No need to do such big and strange overshoot to check two comands.

**Comment by Dominys** [comment]
@MyckolaPetrychko Remove that all getAll methods.

**Pull request merged**